README.md

```diff
@@ -27,36 +27,56 @@ npm install
 grunt install
 ```
 
-You can also use the built version of the SDK included in this repo if you don't want to clone this and build it yourself. It should match the version we maintain on our CDN.
-
 When done and successful, the whole package will be bundled in */dist* directory and available in both minified and un-minified formats as a standalone browserify package.
 
-Configuration
--------------
 Import the SDK on your web page where you want to use Timelines.
 
 ```html
 <script src="js/ls-js-sdk/api.js"></script>
 ```
 
-Once the SDK is imported, it will expose a global LS namespace and it needs to be initialised with unique *app_id* and *app_key* mentioned in the Prerequisites section of this document.
+or for the minified version:
+
+```html
+<script src="js/ls-js-sdk/api.min.js"></script>
+```
+
+You can also use the built version of the SDK included in this repo if you do not wish to clone the repository and build it yourself. It should match the version we maintain on our CDN.
+
+Configuration
+-------------
+Once the SDK is imported, it will expose a global LS namespace and it needs to be initialised. It can be done twofold, depending on a usage scenario:
```

Comment:
'It can be done twofold, depending on a usage scenario" should be, "There are currently two ways to implement the SDK, depending on a usage scenario"